genesyscloud/resource_genesyscloud_knowledge_category.go

```diff
@@ -242,10 +242,27 @@ func deleteKnowledgeCategory(ctx context.Context, d *schema.ResourceData, meta i
 	})
 }
 
-func buildKnowledgeCategory(categoryIn map[string]interface{}) *platformclientv2.Categoryrequest {
+func buildKnowledgeCategoryUpdate(categoryIn map[string]interface{}) *platformclientv2.Categoryupdaterequest {
```

Comment:
The struct `Categoryrequest` is gone and was replaced with one for updates and on for creates. I looked at the schema for each in the go sdk code base and they are the exact same still